tests/Tools/PhpClassComparer.php

```diff
@@ -45,7 +45,7 @@ class PhpClassComparer
      */
     public static function compare(string $phpClassOne, string $phpClassTwo, bool $printDiffs = true): bool
     {
-        $phpParser = (new ParserFactory)->create(ParserFactory::PREFER_PHP7);
+        $phpParser = (new ParserFactory)->createForNewestSupportedVersion();
```

Comment:
I changed these here just to resolve some unittests issues.

Reply:
Merged it with main, this is resolved due to the latest PR. So, reverted this change.